primefaces/src/test/java/org/primefaces/util/FileUploadUtilsTest.java

```diff
@@ -148,10 +148,6 @@ public void isValidTypeContentTypeCheck() throws IOException {
         when(fileUpload.getAccept()).thenReturn("image/tiff");
         Assertions.assertFalse(FileUploadUtils.isValidType(appContext, fileUpload, createFile("test.png", "image/png", png)));
 
-        //Tampered - Apache Tika or Mime Type must be in the classpath for this to work
-        when(fileUpload.getAccept()).thenReturn("image/gif");
-        Assertions.assertFalse(FileUploadUtils.isValidType(appContext, fileUpload, createFile("test.gif", "image/gif", exe)));
-        Assertions.assertTrue(FileUploadUtils.isValidType(appContext, fileUpload, createFile("test.png", "image/png", gif)));
```

Comment:
why removed?

Reply:
will investige this further on monday, removed those because these tests were failing and I assumed no proper FileTypeDetector impl was present on classpath - I only checked for tika and not for mime-types

Reply:
I reverted that change; the test was failing because exposing the file name to mime-types caused it to go the lazy route of looking up the file extension instead of inspecting the file content